### PR TITLE
docs: List packages and insert links to npm

### PR DESCRIPTION
**Pull request by lipis**
## Pull Request Checklist

- [ ] My code is covered by tests
- [ ] I will [merge the PR as breaking change](https://github.com/wireapp/wire-web-packages/wiki/Releases#create-major-release), if the API contract changes

### CODE REVIEW

**Comment by lipis** [comment]
Also for most packages we need to have a small description on what it does. At the moment we have only the instructions on how to run/test in most cases.

**Comment by ffflorian** [comment]
Please rename to `docs: ...`

**Comment by bennyn** [comment]
Good call. I updated the lists and categories. Please have a look, @lipis.

File: README.md [hunk 1]
```diff
@@ -8,7 +8,46 @@ For licensing information, see the attached LICENSE file and the list of third-p
...
+**Common utilities**
+
+- [@wireapp/commons](https://npmjs.com/package/@wireapp/commons)
+- [@wireapp/lru-cache](https://npmjs.com/package/@wireapp/lru-cache)
+- [@wireapp/priority-queue](https://npmjs.com/package/@wireapp/priority-queue)
+- [@wireapp/store-engine](https://npmjs.com/package/@wireapp/store-engine)
+
+**Configuration**
+
+- [@wireapp/copy-config](https://npmjs.com/package/@wireapp/copy-config)
```
Comment (ffflorian) [inline README.md]: This is actually an internal tool, too.

Comment (bennyn) [reply README.md]: changed

File: README.md [hunk 2]
```diff
@@ -8,7 +8,48 @@ For licensing information, see the attached LICENSE file and the list of third-p
...
+- [@wireapp/cbor](https://npmjs.com/package/@wireapp/cbor)
+- [@wireapp/cryptobox](https://npmjs.com/package/@wireapp/cryptobox)
+- [@wireapp/proteus](https://npmjs.com/package/@wireapp/proteus)
+
+**Internal tools**
+
+- [@wireapp/copy-config](https://npmjs.com/package/@wireapp/copy-config)
+- [@wireapp/license-collector](https://npmjs.com/package/@wireapp/license-collector)
+
+**User Interface**
```
Comment (ffflorian) [inline README.md]:
```suggestion
**User interface**
```

Comment (ffflorian) [reply README.md]: :clown_face:

**Pull request merged**